frontend/src/components/main/Main.js

```diff
@@ -226,9 +228,23 @@ export default function Main(props) {
       { vaultStore.balanceOf.value>0 &&
       <div className="main-container">
         <div className="element">
-            <label className="paste-label" style={{textAlign: 'center', width: "100%"}}>Your balance: 
+            <label className="paste-label" style={{textAlign: 'center', width: "100%"}}>Your balance:
             <span style={{color: 'green'}}>{decimalFormat(vaultStore.balanceOf.value, vaultStore.decimals)}</span></label>
           </div>
+          <div className="row main-container">
+            <div class="col-6">
+            <div className="element">
+            <label className="paste-label" style={{textAlign: 'center', width: "100%"}}>Your balance Weth:
+            <span style={{color: 'green'}}>{dinamicFixed(decimalFormat(dinamicFixed(vaultStore.userToken[0], 0), tokenStore.decimalsEth),4)}</span></label>
```

Comment:
Because if userToken have some decimals number, but the currency doesnt allow more decimals
(example, eth have 18 dec, if i have a number of 18 AND other decimals after comma, this will throw an error)
so i need to use the first dinamicFixed, the others one its just some pretty formatting